src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/RazorFormattingTest.cs.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
Lol while you're changing this lets retire the .cs.cs ending 🤣

Reply:
Whoops, knew I forgot something!